src/services.js

```diff
@@ -70,6 +70,15 @@ export function activityForDoc(nodeOrDoc) {
 }
 
 /**
+ * @param {!Node|!./service/ampdoc-impl.AmpDoc} nodeOrDoc
+ * @return {!../extensions/amp-form/0.1/amp-form.AmpFormService}
+ */
+export function ampFormServiceForDoc(nodeOrDoc) {
+  return /** @type {!../extensions/amp-form/0.1/amp-form.AmpFormService} */ (
```

Comment:
This should be `getElementServiceForDoc`, right? It requires an extension. And thus, the return value should be a promise.